bigtable/benchmarks/endurance_benchmark.cc

```diff
@@ -147,42 +147,29 @@ long RunBenchmark(bigtable::benchmarks::Benchmark& benchmark,
   long total_ops = 0;
 
   BenchmarkResult partial = {};
-  // We pre-allocate the buffer based on largely a guess, at 2.5ms per call, a
-  // run of 30 minutes produces 400 * 15 * 60 elements, allocate twice as much
-  // because why not?
-  partial.operations.reserve(kPartialResultsPeriod * 60 * 400);
```

Comment:
The build is failing because you no longer use `kPartialResultsPeriod`, please remove this constant from the code.  In general, I expect you to follow up with any build breakage and fix those problems.

Reply:
Fixed it, is there any way to check auto build.
My end build ran successfully, using make all.

Reply:
The CI builds are available from:

https://travis-ci.org/GoogleCloudPlatform/google-cloud-cpp
https://ci.appveyor.com/project/coryan/google-cloud-cpp

The builds also run on each pull request and are linked from GitHub, look at #400 where it says "all checks passed" for an example.  There are instructions in CONTRIBUTING.md to run the builds locally for different Linux distributions, compilers, and compile options.  And there are links there on how to setup a local build for Windows too.